app/assets/javascripts/miq_application.js

```diff
@@ -1540,7 +1548,7 @@ function miqToolbarOnClick(_e) {
       tb_url += "/" + ManageIQ.record.recordId;
     }
     tb_url += "?pressed=";
-    if (typeof button.data('pressed') == "undefined") {
+    if (typeof button.data('pressed') == "undefined" && button.data('click')) {
```

Comment:
Thanks, either that, or a 3-way `if`, with the pressed undefined and no click branch doing nothing..